wallet/src/de/schildbach/wallet/ui/DecryptSeedWithPinDialog.kt

```diff
@@ -15,49 +16,50 @@ import org.bitcoinj.wallet.DeterministicSeed
  */
 @AndroidEntryPoint
 class DecryptSeedWithPinDialog(
-    private var onSuccessOrDismiss: ((DeterministicSeed?) -> Unit)?
+    private var onSuccessOrDismiss: ((Array<String>) -> Unit)?
```

Comment:
This is an interesting change.  We must not need the `DeterministicSeed` object, only the phrase.